PocketKit/Sources/PocketKit/MyList/SortFilterMenu/SortMenuViewModel.swift

```diff
@@ -0,0 +1,44 @@
+import UIKit
+import Sync
+import Analytics
+
+
+class SortMenuViewModel {
+
+    typealias Snapshot = NSDiffableDataSourceSnapshot<SortSection, SortOption>
+    @Published
+    private var _snapshot = Snapshot()
+    var snapshot: Published<Snapshot>.Publisher { $_snapshot }
```

Comment:
In other parts of the code we have to follow this pattern because we have to conform to a protocol (which cannot require `@Published` properties).

In this case, there's no protocol, so we can simplify down to this:
```swift
@Published
private(set) var snapshot: Snapshot
```

Reply:
Thank you.